resources/styles.css

```diff
@@ -179,3 +179,15 @@ input[type='checkbox'] {
 .mat-mdc-tab:not(.mdc-tab--stacked) {
   @apply h-9 !important;
 }
+
+gn-ui-copy-text-button div input {
+  @apply bg-white rounded-l !important;
```

Comment:
Is there another way to achieve the desired styling? I don't think we should use `!important`.
Maybe you could extent the component class and change the styling?

Reply:
As discussed, keeping it as is for now, but adding a comment to remind us to remove when the component will be extended here, as it should happen as the api form evolves).